lib/glimesh_web/live/user_settings/components/channel_statistics_live.html.leex

```diff
@@ -0,0 +1,28 @@
+<div class="card">
+
+	<table class="table">
+		<thead>
+			<tr>
+				<th>Stream title</th>
+				<th>Stream started</th>
+				<th>Stream ended</th>
+				<th>Category</th>
+				<th>Viewers</th>
+			</tr>
+		</thead>
+
+		<%= for stream <- Glimesh.Streams.list_streams(@channel) do %>
+
+			<tr>
+
+				<td><%= stream.title %></td>
+				<td><%= stream.started_at %></td>
+				<td><%= stream.ended_at %></td>
+				<td>ID: <%= stream.category_id %></td>
```

Comment:
I'd recommend using the category's name instead of the ID as users have no way of converting the ID to a name from the frontend.

You can use the `Glimesh.ChannelCategories.get_category_by_id!/1` function to get a category object from the ID, from there you're able to grab the name ;)

Reply:
Leaving this to add in a later PR on it's own once the rest of it has been approved!